src/common/ios/simulatorPlist.ts

```diff
@@ -46,7 +47,7 @@ export class SimulatorPlist {
                         throw new Error(`Unable to find plist file for ${bundleId}`);
                     } else if (plistCandidates.length > 1) {
                         TelemetryHelper.sendSimpleEvent("multipleDebugPlistFound");
-                        Log.logWarning("Multiple plist candidates found. Application may not be in debug mode");
+                        Log.logWarning(ErrorHelper.getWarning("Multiple plist candidates found. Application may not be in debug mode."));

```

Comment:
Should logWarning call getWarning internally? Or are there cases where we'd call one without the other?